eth/filters/filter_system.go

```diff
@@ -351,6 +376,15 @@ func (es *EventSystem) handleTxsEvent(filters filterIndex, ev core.NewTxsEvent)
 	}
 }
 
+func (es *EventSystem) handleFinalizedEvent(filters filterIndex, ev core.ChainEvent) {
```

Comment:
Because the following event can have the same finalized block with previous one, we don't want to send a new response in this case. We need a variable to keep track last notified finalized block

Reply:
Thanks, I fixed it. Please have a look!